routes/movies-routes.js

```diff
@@ -0,0 +1,79 @@
+// starter code in both routes/celebrities.routes.js and routes/movies.routes.js
+const router = require("express").Router();
+
+const Movie = require('./../models/Movie.model')
+
+/* GET Movies page */
+router.get("/movies/create", (req, res) => res.render("./../views/movies/new-movie"))
```

Comment:
```suggestion
router.get("/movies/create", (req, res) => {
   Celebrity.find().then(allCelebs => {
      res.render("./../views/movies/new-movie", {allCelebs})
   })
})
```
Así puedes ofrecer la lista de famosos al crear.